source/replication/site-replication-overview.rst

```diff
@@ -0,0 +1,462 @@
+- Creation and deletion of service accounts (except those owned by the ``root`` user)
+- Changes to bucket features, such as
+  
+  - :ref:`Policies <minio-policy>`
+  - Tags (for example, see :mc-cmd:`mc tag set`)
+  - :ref:`Object-Lock <minio-object-locking>` configurations (including retention and legal hold configurations)
+  - :ref:`Encryption configuration <minio-encryption-overview>`
+
+Site replication enables bucket versioning for all new and existing buckets on all replicated sites.
+
+
+What doesn't replicate across sites?
+-------------------------------------
+
+The following bucket features don't replicate across sites set up for site replication:
+
+- Notification configurations
+- Lifecycle management (ILM) configurations
+
+This allows users who access different sites to define separate notification or ILM settings.
+
+
+Order of operations
+-------------------
+
+After enabling site replication, identity and access management (IAM) settings sync in the following order:
+
+#. Policies
+#. User accounts (for local users)
+   
+   Note: Sites that use OIDC and LDAP don't sync, and they may not have local users.
+
+#. Groups 
+   
+   For MinIO IDP and LDAP managed groups
+
+#. Service accounts
+
+   Only static accounts with valid claims.
+   Service accounts for ``root`` don't sync.
+
+#. Policy mapping for synced user accounts
+
+#. Policy mapping for `Short Token Service (STS) users <https://docs.min.io/docs/minio-sts-quickstart-guide.html>`
+
+Site healing
+------------
+
+With site replication enabled, bucket metadata and IAM entries can heal from whichever peer site has the most updated entry.
+
+
+Prerequisites
+-------------
+
+- Only *one* site can have data at the time of setup.
+  The other sites must be empty of buckets and objects.
+
+  After configuring site replication, the data replicates to the other sites.
+  Objects written to any site after setting up replication then replicate to all sites.
+
+- All sites must have the same deployment credentials (for example, ``MINIO_ROOT_USER``, ``MINIO_ROOT_PASSWORD``)
+- All sites must use the same external IDP(s), if any
+- For :ref:`SSE-S3 <minio-encryption-sse-s3>` or :ref:`SSE-KMS <minio-encryption-sse-kms>` encryption via KMS, all sites must have access to a central KMS deployment. 
+  This can be achieved via a central KES server or multiple KES servers (say one per site) connected via a central KMS (Vault) server.
+
+
+Configure site replication
+--------------------------
+
+.. tab-set::
+
+   .. tab-item:: Console
+
+      #. Deploy three or more separate MinIO sites, using the same Identity Provider for each site
+
+         Only one site can have any buckets or objects on it.
+         The other sites must be empty.
+
+      #. In a browser, access the Console for one of the sites
+
+         For example, ``https://<addressforsite>:9000``
+         
+         If one of the sites has content, log in to the Console for that site.
```

Comment:
'Tis a pity we have to be specific, but it's good cya to say "replace ``addressforsite`` with the URL or hostname for your MinIO deployment"